README.md

```diff
@@ -39,6 +39,61 @@ Read more about the Ansible Service Broker and Ansible Playbook Bundles in this
 * [Ansible Service Broker - Design](docs/design.md)
 * [Other Documentation](docs/README.md)
 
+## Published Images
+The ansible-service-broker community publishes images in (dockerhub)[https://hub.docker.com/u/ansibleplaybookbundle].
+Image tags:
+- **canary** - The newest source build
+- **latest** - The newest release build
+- **<release_number>** - The stable release of an image
+
+# QuickStart - Running Ansible Service Broker
+
+## Running
+
+The below will use `oc cluster up` to bring up a cluster with Ansible Service Broker installed.
+The script will run from start to finish in under 2 minutes, giving you an easy way to see the Ansible Service Broker in action.
+
+  1. Ensure that your system is setup to run `oc cluster up`
+      * Follow [these instructions](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) if you haven't already seen success with `oc cluster up`
+  1. Download [run_latest_build.sh](https://raw.githubusercontent.com/openshift/ansible-service-broker/master/run_latest_build.sh)
+
+              wget https://raw.githubusercontent.com/openshift/ansible-service-broker/master/run_latest_build.sh
+              chmod +x ./run_latest_build.sh
+
+  1. Execute [run_latest_build.sh](https://raw.githubusercontent.com/openshift/ansible-service-broker/master/run_latest_build.sh)
+      * ./run_latest_build.sh
+          * Will take ~90 seconds
```

Comment:
I would move the 90 seconds line up to the bullet item, like so:

 1. Execute [run_latest_build.sh](https://raw.githubusercontent.com/openshift/ansible-service-broker/master/run_latest_build.sh), this will take ~90 seconds.
     * `./run_latest_build.sh`